sandbox/handlers/nhsapp_accounts.js

```diff
@@ -16,29 +16,37 @@ export async function user_details(req, res, next) {
   }
 
   if (!req.query || !req.query['ods-organisation-code']) {
-    sendError(res, 400, 'ods-organisation-code not provided.')
+    sendError(res, 400, 'Missing ODS Code')
```

Comment:
Have these been changed to get consistency with the messages we will be returning from the BE?

Reply:
Correct.